.github/workflows/ci.yaml

```diff
@@ -0,0 +1,91 @@
+name: "CI"
+
+# Triggered for commits/PRs on master branch
+# + new tags
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master 
+  create:
+    tags:
+      - v*
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  # Cancel previous actions
+  cleanup:
```

Comment:
Actually I find this name confusing, I thought at first it is cleaning up some cache or smth. Maybe we could call it "cancel"?